manager/integration/tests/test_node.py

```diff
@@ -2663,16 +2663,21 @@ def test_disk_eviction_with_node_level_soft_anti_affinity_disabled(client, # NOQ
             break
         time.sleep(common.RETRY_EXEC_INTERVAL)
     assert len(os.listdir(replica_path)) > 0
+    volume = common.wait_for_volume_healthy(client, vol_name)
 
     # Step 7
     replica_count = 1
-    volume = client.by_id_volume(vol_name)
-    volume = volume.updateReplicaCount(replicaCount=replica_count)
+    volume.updateReplicaCount(replicaCount=replica_count)
     volume = common.wait_for_volume_healthy(client, vol_name)
 
-    for r in volume.replicas:
-        if r.hostId != lht_hostId:
-            volume.replicaRemove(name=r.name)
+    for i in range(RETRY_COUNTS):
+        for r in volume.replicas:
+            if r.hostId != lht_hostId:
+                try:
+                    volume.replicaRemove(name=r.name)
+                except Exception:
+                    pass
+                time.sleep(RETRY_INTERVAL)
```

Comment:
Should check if the volume replicas are deleted to break out of the retry loop.

Reply:
Yes. Thank you for pointing this out! Addressed